pkg/terraform/exec/plugins/Gopkg.toml

```diff
@@ -0,0 +1,33 @@
+ignored = [
+  "github.com/libvirt/libvirt-go",
+  "github.com/hashicorp/terraform/*"
+]
+
+[prune]
+  go-tests = true
+  unused-packages = true
+  non-go = true
+
+[[constraint]]
+  name = "github.com/dmacvicar/terraform-provider-libvirt"
+  version = "0.5.1"
+
+[[constraint]]
+  name = "github.com/terraform-providers/terraform-provider-aws"
+  version = "1.52.0"
+
+[[constraint]]
+  name = "github.com/terraform-providers/terraform-provider-ignition"
+  version = "1.0.1"
+
+[[constraint]]
+  name = "github.com/terraform-providers/terraform-provider-openstack"
+  version = "1.12.0"
+
+[[override]]
+  name = "github.com/coreos/ignition"
+  version = "v0.23.0"
+
+[[override]]
+  name = "github.com/gophercloud/gophercloud"
+  revision = "0719c6b22f30132b0ae6c90b038e0d50992107b0"
```

Comment:
Can we drop this and use the version vendired in our package root?  Or is that too complicated or broken?

Reply:
The goal is to keep all the `pkg/terraform/exec` dependencies separate from installer's codebase. 

- terraform vendor requires a very old version of aws and openstack plugins to maintain backward compatible when plugins were emdedded in terraform.
- and the plugins we use require newer versions. that why they are setup this way.